spec/hipchat_api_v2_spec.rb

```diff
@@ -379,6 +379,15 @@
     end
   end
 
+  describe "#delete_room" do
```

Comment:
Can you also add a test for when the room is not found? To verify the error is handled properly.